experimental/adaptive-cards-2/Catering/Controllers/HomeController.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Catering.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Catering.Controllers
+{
+    public class HomeController : Controller
+    {
+        public IConfiguration Configuration { get; set; }
+
+        public HomeController(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+        
+        public async Task<IActionResult> Index()
+        {
+            var secret = Configuration.GetSection("ClientDirectLineSecret")?.Value;
+            var endpoint = Configuration.GetSection("ClientDirectLineEndpoint")?.Value;
+
+            var userId = "dl_" + Guid.NewGuid().ToString();
+            
+            HttpClient client = new HttpClient();
+            HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Post, $"{endpoint}/tokens/generate");
+            request.Headers.Authorization = new System.Net.Http.Headers.AuthenticationHeaderValue("Bearer", secret);
+            request.Content = new StringContent(JsonConvert.SerializeObject(new
+                {
+                    User = new { Id = userId }, 
+                    TrustedOrigins = new string[] 
```

Comment:
I believe this should be updated with placeholder values, or potentially pulled in via configuration?

Reply:
Ah, I actually don't need these because there is no oauth login as we removed the email feature from this sample.

Reply:
So I removed the trusted origins parameter.